### DISCUSSION TITLE
Which permissions required to mark PO line as received?

### DISCUSSION THREAD

**Discussion by miggland** [inventree/InvenTree#3521] [Q&A] [answered]
I want my guy in the warehouse to be able to mark order lines as received, but not do everything in InvenTree (lock it down, less potential for mistakes ;))

However, I've given his user these permissions without the user being allowed to mark lines as received:
![image](https://user-images.githubusercontent.com/1314853/184329505-c5c0b39a-41fa-47e1-a83c-1fdf02f6a456.png)

The message is:
![image](https://user-images.githubusercontent.com/1314853/184329663-d2e1c274-7b83-4603-9aba-18427b938352.png)

Which create permission is required here?

**Comment by eeintech** [comment, marked as answer]
I would open a new issue, you theoretically have the correct permission set (Change for PO).
